auth/src/main/java/com/firebase/ui/auth/data/model/FirebaseUiException.java

```diff
@@ -1,33 +1,46 @@
 package com.firebase.ui.auth.data.model;
```

Comment:
Should we move this to the root package or leave it as is?

Reply:
Since this is `LIBRARY_GROUP` only that's not a breaking change, so if you want to move it to the root package that SGTM.